modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/src/main/resources/META-INF/resources/js/ImagePins.js

```diff
@@ -311,7 +312,11 @@ const ImagePins = ({
 				cont.append('circle')
 					.attr('fill', () => 'transparent')
 					.attr('r', () => addNewPinState.radius)
-					.attr('stroke', () => `#${addNewPinState.fill}`)
+					.attr('stroke', () =>
+						showTooltip.tooltip
+							? '#ffa500'
```

Comment:
For readability, I would recommend using variable at the top of the file describing what this color is.

```js
const COLOR_ORANGE = '#ffa500';

//...

.attr('stroke', () =>
	showTooltip.tooltip
		? COLOR_ORANGE
```

Reply:
Thank you @bryceosterhaus. Yesterday we're discussing about making it complementary to the primary color chosen or to let it be only orange. that's why i leave it like this

Reply:
@garofalog yeah that is fine. My point is that it is going to be easier for future developers to read through this file is we set it as named variable first. It has nothing to do with how the code functions, but is purely for readability within this file.

A future dev will come to this file and read this hex but not understand what it actually is, by reading is as COLOR_ORANGE, they will at least be able to have some helpful context of what this color is.

Reply:
it's fine to me. accoding to [this conversation](https://liferay.slack.com/archives/G9CMBBA9H/p1631092769072200) we'll leave it with orange and we'll add the color-choice-feature in future